recipe/meta.yaml

```diff
@@ -19,6 +19,9 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
```

Comment:
The bot adds this. However the same requirements (down to the formatting) are listed below

https://github.com/regro-cf-autotick-bot/memray-feedstock/blob/ebb575dfe8fea8ac743feef872a382d637bc4040/recipe/meta.yaml#L28-L30

Seems the bot is just having trouble picking up on this

Let's go ahead and close so the bot marks this package as migrated. The then bot can move on to downstream packages as needed